endpoints/openrtb2/auction.go

```diff
@@ -548,7 +548,7 @@ func (deps *endpointDeps) validateImp(imp *openrtb2.Imp, aliases map[string]stri
 		return []error{fmt.Errorf("request.imp[%d] must contain at least one of \"banner\", \"video\", \"audio\", or \"native\"", index)}
 	}
 
-	if err := validateBanner(imp.Banner, index); err != nil {
+	if err := validateBanner(imp.Banner, imp, index); err != nil {
```

Comment:
`ValidateBanner` expects imp index to be the second argument and the third argument needs to be a boolean flag that you'll have to create before calling this function. You can set the value of that boolean flag based on the value of `imp.Instl` and then pass that boolean flag to this function

Reply:
thanks :)

Reply:
@mansinahar think I finally got it. Let me know though if there are more changes needed!